src/build.py

```diff
@@ -112,15 +112,25 @@
 
 # Sync OCaml from a cached tar file because the upstream repository is only
```

Comment:
OCaml hasn't changed, and the comment is still correct. But maybe we could be more clear what it applies to by moving `WASM_STORAGE_BASE` before it or something.